SimpleOAuth/SimpleOAuth/ConfigurationModel/SimpleOAuthConfiguration.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace SimpleOAuth.ConfigurationModel
+{
+    public class SimpleOAuthConfiguration
+    {
+        public GoogleOAuthClientConfiguration? GoogleOAuthClientConfiguration { get; set; }
```

Comment:
just curious, why not using the base class?

```suggestion
        public OAuthClientBaseConfiguration? GoogleOAuthClientConfiguration { get; set; }
```

Reply:
Hey Thomas, the reason why we can move this out is that this would allow us to create a single Extension method to add all the configuration